src/Tests/dotnet.Tests/OutputPathOptionTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.Assertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Xunit;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using System.IO;
+using System.Runtime.CompilerServices;
+
+namespace dotnet.Tests
+{
+    public class OutputPathOptionTests : SdkTest
+    {
+        public OutputPathOptionTests(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Theory]
+        [InlineData("build")]
```

Comment:
This seems to have changed dotnet store too, but it isn't reflected in test here.

Reply:
Yes, `dotnet store` is complicated to test and sort of deprecated, so I just skipped testing it here.